### ISSUE TITLE
[Bug] PinDMs layout bugs

### ISSUE THREAD

**Issue by Akitakek** [Vendicated/Vencord#1257]
### Discord Account

akitake_

### What happens when the bug or crash occurs?

DMs list formatting goes crazy and slowly makes it impossible to use.

### What is the expected behaviour?

To just stay in place.

### How do you recreate this bug or crash?

1. Have some pinned DMs.
2. Remove all unpinned DMs from your list with the "x"
3. Profit

### Errors

N/A

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well

**Comment by SladeHazard** [comment]
+1 https://github.com/mwittrien/BetterDiscordAddons/issues/2513

**Comment by DaimeneX** [comment]
I can confirm that the issue still occurs.
There are three odd behaviours that can be noted.
1. If you have pinned DM's and unpinned DM's you can see only a portion of the unpinned ones. A scrollbar is also present, despite the dm's not exceeding the screen space and requiring an overflow. When you scroll down the missing DM's show up as if they had a "display:none" property removed.
2. If you have pinned DM's but no unpinned DM's, the list freaks out and starts showing multiple "No DM's" SVG while also making it impossible to see some of the pinned dm's. They are also misplaced and can show up in the middle of the list for some reason.
3. If you have no DM's opened the list turns into an endless scroll of the before mentioned SVG's. You can only scroll up and the program will automatically scroll you back down.

I have no themes installed and I'm using the live (not PTB or Canary) build of Discord.

**Comment by SladeHazard** [comment]
I managed to fix the issue by clearing my data but after a while it started
happening again so it's probably because you are storing the user ids
incorrectly and JavaScript rounds them maybe?

**Comment by sapphonie** [comment]
Can reproduce, here's debug info:
```
Vencord Debug Info
Vencord: v1.6.7 • f1f0da4 (Standalone) - 8 Feb 2024
Discord Branch: stable
Client: Discord Desktop v1.0.9032
Platform: Win32
Outdated: false
OpenAsar: false
Last Crash Reason: N/A

Enabled Plugins (40):
CallTimer, CrashHandler, Experiments, FakeNitro, FavoriteGifSearch, FixSpotifyEmbeds, ForceOwnerCrown, MessageLogger, NoBlockedMessages, NoDevtoolsWarning, NoUnblockToJump, OpenInApp, PermissionsViewer, petpet, PinDMs, PlatformIndicators, PronounDB, RelationshipNotifier, RoleColorEverywhere, SecretRingToneEnabler, ShowHiddenChannels, SilentTyping, StartupTimings, TypingIndicator, TypingTweaks, ValidUser, VolumeBooster, WhoReacted

BadgeAPI, ChatInputButtonAPI, CommandsAPI, ContextMenuAPI, MemberListDecoratorsAPI, MessageDecorationsAPI, MessageEventsAPI, NoticesAPI, ServerListAPI, NoTrack, Settings, SupportHelper
```

Happens with or without OpenASAR.

May be related to me switching between multiple accounts.

**Comment by brunoman4006** [comment]
I reinstalled windows a few days ago and after i reinstalled vencord and enabled this plugin it kept doing invisible DMs, yet i don't know if resetting the config file for PinDMs would fix it or not since i don't know where is it located

**Comment by T4M1L0S** [comment]
On my side, my pinned DMs also started to become invisible.

**Comment by Vendicated** [comment]
all fixed

**Comment by SeiferUwU** [comment]
This is still happening, occurs now when you have lots of DMs in different groups but mainly with the Direct Messages area especially if it's collapsible

**Comment by roltairtheprotogen** [comment]
This issue is not fixed, same for #2944 but that thread was locked. Pinned DM's in multiple groups turn invisible unless you scroll further down to reveal them.
Have reported this several times in the Discord server but gotten no response from any staff members. This has been an issue now for at least 2 years, please fix it.
